charts/airbyte-worker/templates/deployment.yaml

```diff
@@ -119,6 +119,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: JOB_KUBE_NAMESPACE
```

Comment:
This is probably a typo and should be JOB_KUBE_SERVICE_ACCOUNT right?

Reply:
This PR has been migrated, @mjstel if you can please re-add your suggestion to the new PR so your authorship is preserved: https://github.com/airbytehq/airbyte-platform/pull/177/files#r1120705574